Package.swift

```diff
@@ -0,0 +1,104 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "maplibre-navigation-ios",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v14)],
+    products: [
+        // Products define the executables and libraries a package produces, making them visible to other packages.
+        .library(
+            name: "MapboxNavigation",
+            targets: [
+                "MapboxNavigation",
+                "MapboxNavigationObjC"
+            ]
+        ),
+        .library(
+            name: "MapboxCoreNavigation",
+            targets: [
+                "MapboxCoreNavigation",
+                "MapboxCoreNavigationObjC"
+            ]
+        ),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/tomasharkema/mapbox-directions-swift.git", branch: "main"),
```

Comment:
Hier ook changes op gedaan die we kunnen reviewen?

Reply:
ja gooi ik er ook ff in!